internal/security/anchore.go

```diff
@@ -204,39 +276,54 @@ func RemoveAnchoreUser(orgId uint, clusterId string) {
 	} else {
 		logger.Infof("Anchore user secret %v deleted.", anchorUserName)
 	}
+	if checkAnchoreUser(anchorUserName, http.MethodDelete) != http.StatusNoContent {
+		logger.Errorf("Error deleting Anchore user: %v", anchorUserName)
+	}
+	logger.Debugf("Anchore user %v deleted.", anchorUserName)
+	if deleteAnchoreAccount(anchorUserName) != http.StatusNoContent {
+		logger.Errorf("Error deleting Anchore account: %v", anchorUserName)
+	}
+	logger.Debugf("Anchore account %v deleting.", anchorUserName)
 }
 
-func MakeAnchoreRequest(orgId uint, clusterId string, method string, url string, body interface{}) (*http.Response, error) {
+// MakeAnchoreRequest do anchore api call
```

Comment:
I would rename this to `DoAnchoreRequest`, Do* naming is some kind of convention in HTTP client libs.